src/plugins/maps_legacy/public/common/opensearch_maps_client.js

```diff
@@ -21,7 +21,7 @@ export class OpenSearchMapsClient extends EMSClient {
       result = await this._fetchWithTimeout(this._manifestServiceUrl);
     } catch (e) {
       // silently ignoring the exception and returning false.
-      return false;
+      return true;
```

Comment:
Yes, it is expected. When `await this._fetchWithTimeout(this._manifestServiceUrl)` fail to access our maps service(no-internet condition), it will make OpenSearchMapsClient isn't enabled, and down to use EMS client [here](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/maps_legacy/public/map/service_settings.js#L130-L147),  in this case, we can actually still use OpenSearchMapsClient rather than EMS client. And actually all our use cases can use OpenSearchMaps client.